.github/workflows/builds.yml

```diff
@@ -16,7 +16,7 @@ on:
 
   # for debugging
```

Comment:
minor: @hugomrdias since you're in this file, can you remove:

`# TODO: bring back release hook (vs on PR)`